src/test/java/uk/gov/ons/fwmt/job_service/service/impl/CSVParsingServiceImplTest.java

```diff
@@ -58,8 +58,10 @@ public void setFromCSVColumnAnnotations() throws IOException {
     //When
     csvParsingServiceImpl.setFromCSVColumnAnnotations(testIngestData, csvParser.iterator().next(), "GFF");
 
+
     //Then
     assertEquals(expected, testIngestData.getTla());
+    assertEquals("testLcfIncentive", testIngestData.getLcfIncentive());
```

Comment:
This check is incorrect. Even the assert above that is incorrect.

Add the new field in the sample_GFF_2018-05-17T15-34-00Z file and sample_LFS_2018-05-17T15-34-00Z and check whether you are getting that value back.